contracts/WstETHPriceFeed.sol

```diff
@@ -2,35 +2,36 @@
 pragma solidity 0.8.15;
 
 import "./vendor/@chainlink/contracts/src/v0.8/interfaces/AggregatorV3Interface.sol";
-
-interface IWstETH {
-    function decimals() external view returns (uint8);
-    function tokensPerStEth() external view returns (uint256);
-}
+import "./IWstETH.sol";
 
 contract WstETHPriceFeed is AggregatorV3Interface {
     /** Custom errors **/
     error InvalidInt256();
-    error NotImplemented();
-
-    string public constant override description = "Custom price feed for wstETH / USD";
 
+    /// @notice Version of the price feed
     uint public constant override version = 1;
 
-    /// @notice Scale for returned prices
+    /// @notice Description of the price feed
+    string public constant override description = "Custom price feed for wstETH / ETH";
+
+    /// @notice Number of decimals for returned prices
     uint8 public override decimals = 8;
```

Comment:
Rather than hardcoding this, we could also make this an immutable set via the constructor.

Reply:
Probably, maybe, since we are passing it in the others?

Reply:
Done